cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -2411,12 +2429,19 @@ def get_xblock_problem(label):
             problem_templates = self.get_templates_of_type('problem')
             return self.get_template(problem_templates, label)
 
-        def verify_openassessment_present(support_level):
+        def verify_openassessment_not_present():
```

Comment:
Why do we need this function? Can we remove this `not_present` check?

Reply:
I just wanted to make sure that we don't have `ORA` under `Problems`, is it not required?

Reply:
No need actually, you can just test for the `staffgradedxblock`.

Reply:
Sure @giovannicimolin  will remove it :)